qemu/tests/expose_host_mtu.py

```diff
@@ -0,0 +1,101 @@
+import logging
+
+from avocado.utils import process
+from virttest import error_context
+from virttest import utils_net
+from virttest import utils_test
+from virttest import env_process
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Expose host MTU to guest test
+
+    1) Boot up guest with param 'host_mtu=4000' in nic part
+    2) Disable NetworkManager in guest
+    3) set mtu of guest tap (eg: tap0) and physical nic (eg: eno1) to
+       4000 in host
+    4) check the mtu in guest
+    5) ping from guest to external host with packet size 3972
+
+    :param test: kvm test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment.
+    """
+
+    def get_ovs_ports(ovs):
+        cmd = "ovs-vsctl list-ports %s" % ovs
+        return process.system_output(cmd).decode()
+
+    def is_ovs_backend(netdst):
+        if netdst in process.system_output("ovs-vsctl list-br",
+                                           ignore_status=True,
+                                           shell=True).decode():
+            return 1
```

Comment:
Why not directly `return netdst in process.system_output("ovs-vsctl list-br", ignore_status=True, shell=True).decode()` or
```
ovs_list = process.system_output("ovs-vsctl list-br", ignore_status=True, shell=True).decode():
return netdst in ovs_list
```

Reply:
I only need to confirm it's ovs backend. I don't care how many or which bridges they are.

Reply:
Oops, I misunderstood what you mean. But I think both methods are OK.

Reply:
@xiywang Sure, I prefer return bool value directly instead of judge it and translate to int, which seems a lower efficient way to me. But I am also okay with current approach.